flatpak/com.matthiasnehlsen.lotti.local.yml

```
@@ -0,0 +1,71 @@
+app-id: com.matthiasnehlsen.lotti
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: '24.08'
+    add-ld-path: .
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)